crates/swc_ecma_minifier/tests/fixture/issues/vercel/004/output.js

```diff
@@ -1,9 +0,0 @@
-function ItemsList() {
```

Comment:
This was a test about dependency issue of sequential inliner and I think we should not remove the test

Reply:
~~It's not removed, it's just compressed~~

Should we add an export here?

Reply:
I think it will work